packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -6,6 +6,28 @@ import { spawnSync, SpawnSyncOptions } from 'child_process';
  * @experimental
  */
 export interface BundlingOptions {
+  /**
+   * Docker bundling
+   */
```

Comment:
add some commentary about the fact that this is required in order to allow bundling to happen in any environment, and also mention that we recommend to also provide a `local` bundling method.